crates/env/src/engine/on_chain/ext.rs

```diff
@@ -375,7 +381,8 @@ mod sys {
             output_len_ptr: Ptr32Mut<u32>,
         ) -> ReturnCode;
 
-        pub fn seal_set_storage(
+        #[link_name = "seal_set_storage"]
```

Comment:
We need to do this since that's how the Contract pallet exports the `seal1` function, right? Just thinking if this can cause any sort of naming conflicts if we mix `seal0` and `seal1` methods within an ink! contract

Reply:
We import both `seal0` and `seal1` versions of the function, which have the same name in both modules, but this shouldn't be an issue in ink! once they are linked to differently named rust functions.

Reply:
double-checked, and yes, you can use both within the same ink! contract - that's the whole point ;)
```rust
        #[ink(message)]
        pub fn seal0(&self) {
            ink_env::set_contract_storage(&Default::default(), &1)
        }

        #[ink(message)]
        pub fn seal1(&self) -> Option<u32> {
            ink_env::set_contract_storage_inform(&Default::default(), &1)
        }
```